README.md

```diff
@@ -43,9 +43,15 @@ If you are choosing between using
 [mongos](https://docs.mongodb.com/manual/reference/program/mongos/) and
 using mongo shard with `mongo_api` - prefer mongos and use `mc_worker_api`.
 
+By default, this driver works only with MongoDB version 5.0 and below as it uses
```

Comment:
can we make the modern one by default?
or making it to decide itself by detecting versions?

Reply:
The README has been updated.